bindings/rust/s2n-tls-sys/README.md

```diff
@@ -1,3 +1,30 @@
 This crates provides low level rust bindings for [s2n-tls](https://github.com/aws/s2n-tls) which are autogenerated with [bindgen](https://github.com/rust-lang/rust-bindgen)
 
-This crate is not intended for direct consumption by end consumers. Interested developers should instead look at the [s2n-tls](https://crates.io/crates/s2n-tls) or [s2n-tls-tokio](https://crates.io/crates/s2n-tls-tokio) crates. These  provide higher-level, more ergonomic bindings than the `s2n-tls-sys` crate.
\ No newline at end of file
+This crate is not intended for direct consumption by end consumers. Interested developers should instead look at the [s2n-tls](https://crates.io/crates/s2n-tls) or [s2n-tls-tokio](https://crates.io/crates/s2n-tls-tokio) crates. These  provide higher-level, more ergonomic bindings than the `s2n-tls-sys` crate.
+
+This crate supports using pre-built libs2n following the next steps below:
+
+1. Clone [s2n-tls](https://github.com/aws/s2n-tls) and compile your preferred configuration of s2n-tls. 
+
+You may choose to link against a specific libcrypto at this step. For more information, see [Building with a specific libcrypto](https://github.com/aws/s2n-tls/blob/main/docs/BUILD.md#building-with-a-specific-libcrypto)
+```
+cmake . -Bbuild -DBUILD_SHARED_LIBS=on -DBUILD_TESTING=off
+cmake --build build -- -j $(nproc)
+```
+
+2. cd into your rust project and set environment variables to libs2n library sources. 
+
+This tells the bindings to link to pre-built libs2n when running the build script for s2n-tls-sys
+```
+export S2N_TLS_LIB_DIR=<PATH_TO_ROOT_OF_S2N_TLS>/build/lib
+export S2N_TLS_INCLUDE_DIR=<PATH_TO_ROOT_OF_S2N_TLS>/api
+export LD_LIBRARY_PATH=$S2N_TLS_LIB_DIR:$LD_LIBRARY_PATH
+```
+
+3. Build your project. This triggers the build script for s2n-tls-sys
```

Comment:
In this step, let's call out what's different vs the default?

Honestly, it would be lovely if you would add a little bit more documentation to this readme about the standard build setup.
- `s2n-tls-sys` contains the raw C code of `s2n-tls`.
- by default it will use the system C compiler to build `libs2n.a` and link it to the rust bindings
- by default it will link against aws-lc through the `aws-lc-rs` crate.

That would probably make the "supply your own lib" discussion a bit easier to follow.

Reply:
I added this information at the beginning section so it flows like 
1. what are the default build setups
2. you can use pre-built libs2n
3. here's how